testing/helpers/nativePointerMock.js

```diff
@@ -483,22 +482,20 @@
                     targetTouches = createTouchListByArray(targetTouches);
                     changedTouches = createTouchListByArray(changedTouches);
 
-                    customEvent.initTouchEvent(type, bubbles, cancelable, view, detail,
-                        screenX, screenY, clientX, clientY,
-                        ctrlKey, altKey, shiftKey, metaKey,
-                        touches, targetTouches, changedTouches,
-                        scale, rotation);
+                    customEvent = new window.TouchEvent(type, {
+                        cancelable: cancelable,
+                        bubbles: bubbles,
+                        touches: touches,
+                        targetTouches: targetTouches,
+                        changedTouches: changedTouches
+                    });
 
                 } else {
                     throw Error('No touch event simulation framework present for iOS, ' + UA.ios + '.');
                 }
             } else if(UA.chrome && ('ontouchstart' in window)) {
-                customEvent = document.createEvent("UIEvent");
-
-                customEvent.initEvent(type, bubbles, cancelable);
```

Comment:
We remove `initEvent` call because it is used to initialize the value of an event created using `document.createEvent()`. See more:
https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent